cdap-docs/_common/tabbed-parsed-literal.py

```diff
@@ -226,6 +228,21 @@ def dequote(s):
         return s[1:-1]
     return s
     
+def clean_ascii(s):
+    """
+    If a string has any non-ASCII characters, replace them with a hyphen.
+    """
+    s_clean = ''
+    for c in s:
```

Comment:
better variable names, perhaps? Although I can sort of guess that this is ``for char in string``?